### PR TITLE
Add Turkish language for Mod Menu Helper

### PR DESCRIPTION
**Pull request by egeesin**
- Translate existing strings to Turkish

Note: May need more source context (based on en_us.json) and/or proofreading on **line 2, 3, 21 and 32** to improve the translation.
Not: Çevirinin iyileştirilmesi için **2, 3, 21 ve 32'nci satırda** biraz daha kontrole ve/veya (en_us.json baz alınarak) kaynak bağlamına ihtiyacı olabilir.

### CODE REVIEW

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:1]: Please rename the file to `tr_tr.json`. Minecraft uses lowercase language codes for a long time, though I can't remember why they actually changed it.

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.dropInfo.line1": "FO modlarında §9✎§r veya §8✎§r, altlarda bulunan: §c*§r;",
+  "modmenu.dropInfo.line2": "diğerleri desteklenmemektedir.",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:3]: Context:

- The mods that are included as part of the Fabulously Optimized modpack have the blue or gray pencil icon before the mod options' description text. 
- Mods that may be used as an alternative by the user or during development of the pack (e.g. AdvancementInfo vs Advancement Enlarger) have a red asterisk before the main description line. 
- Mods that have neither the pencil or asterisk are not a part of Fabulously Optimized.

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.animatica": "Optifine'ın hareketli dokular\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost\n§9✎§8 tuş atamasıyla düzelt.",
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Hızlı yataklar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bettermounthud": "Atın üstündeyken yemek ve XP'yi göster\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.bobby": "Sunuculardaki yüklü yığınları yüklü tutar\n§9✎§8 Varlıkları blokla, yığınları azalt, önbelleği koru, ...",
+  "modmenu.summaryTranslation.borderlessmining": "Tam ekrandayken Alt+Tab desteği (opt-in)\n§9✎§8 Modu, macOS desteğini, özel boyut vb. aç/kapa",
+  "modmenu.summaryTranslation.capes": "Optifine veya diğerlerinden pelerinler\n§9✎§8 Türler, tercih edilen tür, elitra, ...",
+  "modmenu.summaryTranslation.citresewn": "Optifine'ın özel eşyaları\n§9✎§8 Modu aç/kapa, bozuk paketlere izin ver, ...",
+  "modmenu.summaryTranslation.colormatic": "Optifine'ın özel renkleri\n§9✎§8 Gökyüzünü veya yer boşluğunu temizle, ışık yumuşatma",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:15]: The line gets cut off after "ışık"

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.continuity": "Optifine'ın bağlantılı dokuları\n§9✎§8 Modu, metodu, kaynak paketlerini aç/kapa",
+  "modmenu.summaryTranslation.cullleaves": "Optifine'ın \"daha iyi ağaçları\"\n§9✎§8 Modu, kaynak paketini aç/kapa",
+  "modmenu.summaryTranslation.cem": "Optifine'ın özel varlıkları (alfa)\n§9✎§8 Optifine desteği, model düzeltimi",
+  "modmenu.summaryTranslation.dcch": "Farklı dünyalardan yazım geçmişini tut\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.dynamicfps": "Arkaplandayken FPS'i düşür\n§9✎§8 Modu aç/kapa, üzerindeyken normale çevir, ...",
+  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandıklar, tabelalar, yataklar vb.\n§9✎§8 Geliştirilecek bloklar",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:21]: Context: the mod makes chests, signs, beds, bells, shulker boxes render faster. It is possible to toggle each optimization off in options, and choose the method with which the blocks will be optimized.

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.entityculling": "Engellerin arkasındaki mobları görüntüleme\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.entity_texture_features": "Optifine'daki rastgele ve ışık emen moblar\n§9✎§8 Rastgele moblar, ışık emiciliği, göz kırpabilmeleri, ...",
+  "modmenu.summaryTranslation.fabrishot": "F9 ile yüksek çözünürlüklü görüntü al\n§9✎§8 F2 tuşunu, çözünürlüğü, tuş ataması vb. üzerine yaz.",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:24]: The line gets cut off after "üzerine"

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.farsight": "§c*§r Sunuculardaki yüklü yığınları yüklü tutar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.fastchest": "§c*§r Daha hızlı sandıklar\n§9✎§8 Modu aç/kapa",
+  "modmenu.summaryTranslation.fastopenlinksandfolders": "Bağlantı veya klasörleri açarken donmayı önler\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.ferritecore": "RAM kullanımı azaltır\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.forgetmechunk": "DFixes lag spikes on chunk borders\n§8✎ (seçenekleri yok)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:29]: Seems like you forgot to translate

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.hydrogen": "Reduce RAM usage\n§8✎ (seçenekleri yok)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:30]:
```suggestion
  "modmenu.summaryTranslation.hydrogen": "RAM kullanımı azaltır\n§8✎ (seçenekleri yok)",
```

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.iris": "Optifine'ın shaderları\n§9✎§8 Modu aç/kapa, shader, tuş atamaları vb. seç",
+  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmüz çizgileri düzeltir\n§9✎§8 Metod",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:32]: Context: just look at the comparison images on Curseforge: https://www.curseforge.com/minecraft/mc-mods/item-model-fix

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.lambdynlights": "Optifine'ın \"dinamik ışıklandırmaları\"\n§9✎§8 Kalite, varlık seçenekleri, ...",
+  "modmenu.summaryTranslation.languagereload": "Hızlı dil değiştirme, arama çubuğu\n§9✎§8 Dili yeniden yüklemek için F3+J",
+  "modmenu.summaryTranslation.lazydfu": "Daha hızlı oyun açılışı\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.lithium": "Fizikleri ve yığınları optimize eder\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.modmenu": "Bu mod menüsü\n§9✎§8 Tuş yerleşimi, filtreleme, sıralama, ...",
+  "modmenu.summaryTranslation.morechathistory": "Sohbet geçmiş limitini arttırır\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.nofade": "Yüklenirken solma animasyonunu kaldırır\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.notenoughcrashes": "Çökmeden sonra bildir ve oyna\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.phosphor": "§c*§r Işığı optimize eder, daha hızlı yığınlar\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.puzzle": "Bazı Optifine özellikleri\n§9✎§8 Ayarlamak için modu genişlet( ▼ simgesine tıkla)",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:43]:
```suggestion
  "modmenu.summaryTranslation.puzzle": "Bazı Optifine özellikleri\n§9✎§8 Ayarlamak için modu genişlet (▼ simgesine tıkla)",
```

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  "modmenu.summaryTranslation.puzzle-base": "Bazı Optifine özellikleri\n§9✎§8 Kaynak paketinden açılış ekranı, ...",
+  "modmenu.summaryTranslation.smoothboot": "Oyun donmalarını azaltır\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.sodium": "Görüntülemeyi optimize eder, daha iyi FPS\n§9✎§8 Ekran Ayarları'ndan: optimizasyonlar",
+  "modmenu.summaryTranslation.starlight": "Işığı optimize eder, daha hızlı yığınlar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.reeses-sodium-options": "Dikey ekran ayarları\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.sodium-extra": "Optifine'ın grafik seçenekleri\n§9✎§8 Ekran Ayarlar'ından: animasyonlar, parçacıklar, ...",
+  "modmenu.summaryTranslation.tiefix": "Tanılayıcı veri gönderimini devre dışı bırakır, hataları düzeltir\n§9✎§8 Tanıları ve hata çözümlerini aç/kapa",
```
Comment (Madis0) [inline Mod Menu Helper/assets/modmenu/lang/tr_TR.json:50]: Second line is not displayed, moves to a new row after "bırakır,"

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json (hunk 6 above)
Comment (egeesin) [reply Mod Menu Helper/assets/modmenu/lang/tr_TR.json]: Yeah, ironically and coincidentally, I forgot to the translate the description of *Forgot Me* Chunk mod, haha. Thanks for notice!

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json (hunk 8 above)
Comment (egeesin) [reply Mod Menu Helper/assets/modmenu/lang/tr_TR.json]: I understood the description itself, but didn't get the things after line break which directly related to the context on line 2 & 3 you provided. Translation should be OK as it is.

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json (hunk 10 above)
Comment (egeesin) [reply Mod Menu Helper/assets/modmenu/lang/tr_TR.json]: Tried to narrow it down. 👍 You may want to check TieFix mod desc.

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json (hunk 5 above)
Comment (egeesin) [reply Mod Menu Helper/assets/modmenu/lang/tr_TR.json]: Tried to narrow it down. 👍 You may want to check Fabrishot mod desc.

File: Mod Menu Helper/assets/modmenu/lang/tr_TR.json (hunk 3 above)
Comment (egeesin) [reply Mod Menu Helper/assets/modmenu/lang/tr_TR.json]: Tried to narrow it down. 👍 You may want to check Colormatic mod desc.

**Pull request merged**